apps/sched/interface.html

```diff
@@ -185,15 +285,26 @@ <h4>Manage dated events</h4>
     <div class="float-right">
       <button class="btn" onclick="addAlarm()">
         <i class="icon icon-plus"></i>
+        Alarm (with date)
+      </button>
+      <button class="btn" onclick="addDailyAlarm()">
+        <i class="icon icon-plus"></i>
+        Alarm
+      </button>
+      <button class="btn" onclick="addTimer()">
+        <i class="icon icon-plus"></i>
+        Timer
       </button>
     </div>
 
     <table class="table">
       <thead>
         <tr>
-          <th>Date</th>
+          <th>Type</th>
+          <th>Date/Time</th>
           <th>Summary</th>
           <th></th>
+          <th></th>
```

Comment:
I think this extra column could use a caption.
Maybe simply "On?"

Reply:
Sounds good to me 👍